css/index.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  margin: 0 auto;
+  padding: 10px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
```

Comment:
Consider removing vendor prefixes for `display: flex;` as modern browsers have good support for flexbox without prefixes. Use tools like Autoprefixer if you need to support older browsers.